maintainers/maintainer-list.nix

```diff
@@ -16420,4 +16420,10 @@
     github = "srounce";
     githubId = 60792;
   };
+  eliandoran = {
+    email = "contact@eliandoran.me";
+    name = "Elian Doran";
+    github = "eliandoran";
+    githubId = 21236836;
+  };
```

Comment:
I think the entries here should be alphabetically sorted.

Reply:
@FliegendeWurst , good point. Fixed!